db/migrations/20221005105300_create_users_table.sql

```diff
@@ -0,0 +1,15 @@
+-- migrate:up
+CREATE TABLE users (
+id INT NOT NULL AUTO_INCREMENT,
+account_email VARCHAR(50),
+name VARCHAR(20),
+point DECIMAL(10,0) default 50000,
+kakao_id BIGINT UNIQUE,
+profile_image VARCHAR(100),
+is_subscribe BOOLEAN,
+created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+PRIMARY KEY (id)
+);
```

Comment:
- 적절하게 들여씌기 해서 가독성을 높여주면 좋을 것 같습니다.
- 카카오로 받을 수 있는 정보들 확인이 되었나요?? 받을 수 없는 데이터들은 None 또는 default 값을 줄 수 있게 지정해 주세요.
```suggestion
CREATE TABLE users (
  id INT NOT NULL AUTO_INCREMENT,
  account_email VARCHAR(50),
  name VARCHAR(20),
  point DECIMAL(10,0) default 50000,
  kakao_id BIGINT UNIQUE,
  profile_image VARCHAR(100),
  is_subscribe BOOLEAN,
  created_at TIMESTAMP NOT NULL DEFAULT NOW(),
  PRIMARY KEY (id)
);
```